fastlane/Fastfile

```diff
@@ -549,17 +548,46 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each { |source, destination|
+      source_dir = Dir.pwd + "/metadata/" + source
+      Dir.children(source_dir).select { |file| 
+        FileUtils.mv source_dir + "/" + file, Dir.pwd + "/metadata/" + destination + "/" + file
+      }
+      FileUtils.remove_dir(source_dir)
+    }
+
+    # Copy source files into `en-US`
+    enUS_path = Dir.pwd + "/metadata/en-US/"
+
+    FileUtils.mkdir_p(enUS_path) unless File.directory?(enUS_path)
 
-      support_path = Dir.pwd + "/metadata/#{locale}/support_url.txt"
-      write_file(support_path, "https://docs.woocommerce.com/")
+    [
+      "promo_screenshot_1.txt",
+      "promo_screenshot_2.txt",
+      "promo_screenshot_3.txt",
+      "promo_screenshot_4.txt",
+      "promo_screenshot_5.txt",
+    ].each { |source|
+      FileUtils.cp( Dir.pwd + "/appstoreres/metadata/source/" + source, enUS_path + source)
     }
+
```

Comment:
```suggestion
    ].each do |source|
      FileUtils.cp( File.join(Dir.pwd, 'appstoreres', 'metadata', 'source', source), File.join(enUS_path, source) )
    end

```

Note that we could also consider using the fact that `FileUtils.cp` accepts an array as its first argument (source) and will then copy all the files listed in that array into the destination (that has to be a folder in that case). So `FileUtils.cp([list,of,files], enUS_path)` would be enough, except that we would still need to `map` the list of file basenames `promo_screenshot_N.txt` to their full path to produce the array, so probably not worth going that far in refactoring.

Reply:
As part of the oft-aforementioned localization tooling, it'd be nice to also look into where/how to store these resources initially. IMHO it might make sense to have the `default` directory be the source for everything that goes off for translation, which would avoid the need for this entirely.